### PR TITLE
Add additional query time statistics

### PR DESCRIPTION
**Pull request by tanjialiang**
The newly added coordinator level query time statistics provides additional insights on query latency bottlenecks. This is beneficial for performance debugging.

```
== NO RELEASE NOTE ==
```

depends on https://github.com/facebookexternal/presto-facebook/pull/1562

### CODE REVIEW

**Review by shixuan-fan — APPROVED** [review]
LGTM. Nice ASCII art :)

File: presto-spi/src/main/java/com/facebook/presto/spi/eventlistener/QueryStatistics.java [hunk 1]
```diff
@@ -23,9 +23,13 @@
     private final Duration cpuTime;
     private final Duration retriedCpuTime;
     private final Duration wallTime;
-    private final Duration queuedTime;
     private final Duration waitingForPrerequisitesTime;
+    private final Duration queuedTime;
```
Comment (highker) [inline presto-spi/src/main/java/com/facebook/presto/spi/eventlistener/QueryStatistics.java:27]: If we are to swap the order of `waitingForPrerequisitesTime` and `queuedTime`, we probably would do the same for their getters. Or we don't have to swap the order.

**Review by highker — APPROVED** [review]

File: presto-spi/src/main/java/com/facebook/presto/spi/eventlistener/QueryStatistics.java (hunk 1 above)
Comment (tanjialiang) [reply presto-spi/src/main/java/com/facebook/presto/spi/eventlistener/QueryStatistics.java:27]: Good call. Let's swap, making it more intuitive.

**Review by jainxrohit — APPROVED** [review]
Very nice workflow chart!

**Pull request merged**